### PR TITLE
Document Kerberos auth should be enabled with Kerberized Hive.

### PR DESCRIPTION
**Pull request by ebd2**

### CODE REVIEW

**Comment by ebd2** [comment]
@BrendaNoonan, please review when you have a chance. It should be quick :-)

File: presto-docs/src/main/sphinx/security/hive.rst [hunk 1]
```diff
@@ -18,6 +18,16 @@ Lists)` to provide additional security for data.
...
 .. _hive-security-kerberos-support:
 
+.. warning::
+
+  Access to the Presto coordinator should be secured using Kerberos when using
+  Kerberos authentication to Hadoop services. Failure to secure access to the
+  Presto coordinator could result in unauthorized access to sensitive data on
+  the Hadoop cluster.
+
+  :doc:`/security/server`
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/security/hive.rst]: Can you make these links clearer? Something like "See <link> and <link>"

**Comment by cawallin** [comment]
looks good

File: presto-docs/src/main/sphinx/security/hive.rst [hunk 2]
```diff
@@ -18,6 +18,16 @@ Lists)` to provide additional security for data.
 
 .. _hive-security-kerberos-support:
 
+.. warning::
```
Comment (maciejgrzybek) [inline presto-docs/src/main/sphinx/security/hive.rst:21]: Why it's not highlighted / bolded? I don't see this on preview. Maybe GH screwed it?

Comment (ebd2) [inline presto-docs/src/main/sphinx/security/hive.rst:21]: Github doesn't render sphinx stuff correctly :-(

**Comment by maciejgrzybek** [comment]
Shouldn't it be more bold statement? E.g. "if you don't secure access to coordinator, your all Kerberos efforts are pointless". Maybe not that much but you get the point. ;)

**Comment by ebd2** [comment]
Merged.

**Pull request closed without merging**